dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/blob.properties

```diff
@@ -10,6 +10,7 @@ implementation=objectstorage
 # hybrid is using both objectstorage for unfrequently read or big blobs & cassandra for small, often read blobs
 # Size threshold for considering a blob as 'big', causing it to be saved in the low cost blobStore
 # Optional, defaults to 32768 bytes (32KB), must be positive
+# Deprecated
```

Comment:
```
# Mandatory, allowed values are: cassandra, objectstorage, hybrid (deprecated)
# hybrid is using both objectstorage for unfrequently read or big blobs & cassandra for small, often read blobs. 
# Deprecated. Use Blob Store cache instead.
implementation=objectstorage
```

Reply:
> Can we add a @deprecated annotation directly in hybrid blob store class?
> 
> When will it be removed? (3.6.0)

We do have one already?

Reply:
Not yet. We will soon release 3.5.0, 3.6.0 will be release ~ September